homeassistant/components/onewire/sensor.py

```diff
@@ -185,6 +185,36 @@ class OneWireSensorEntityDescription(OneWireEntityDescription, SensorEntityDescr
         ),
     ),
     "28": (SIMPLE_TEMPERATURE_SENSOR_DESCRIPTION,),
+    "30": (
+        SIMPLE_TEMPERATURE_SENSOR_DESCRIPTION,
+        OneWireSensorEntityDescription(
+            key="typeK/temperature",
+            device_class=SensorDeviceClass.TEMPERATURE,
+            entity_registry_enabled_default=False,
+            name="Temperature TypeK",
+            native_unit_of_measurement=TEMP_CELSIUS,
+            read_mode=READ_MODE_FLOAT,
+            state_class=SensorStateClass.MEASUREMENT,
+        ),
```

Comment:
```suggestion
        OneWireSensorEntityDescription(
            key="type/temperature",
            device_class=SensorDeviceClass.TEMPERATURE,
            entity_registry_enabled_default=False,
            name="Temperature Type",
            native_unit_of_measurement=TEMP_CELSIUS,
            override_key="typeK/temperature",
            read_mode=READ_MODE_FLOAT,
            state_class=SensorStateClass.MEASUREMENT,
        ),
```

Reply:
I probably like `key="typeX/temperature"` best, because it matches the OWFS documentation.
Name could be adjusted also, to `TC temperature` or `Thermocouple temperature`, or...